### DISCUSSION TITLE
Fetch activities by specific user

### DISCUSSION THREAD

**Discussion by mira-thakkar** [spatie/laravel-activitylog#744] [Q&A] [answered]
Hello!
I am using this package from so long, and it's a working like a charm to store activity log & create history based on that.

**Comment by Gummibeer** [comment, marked as answer]
Hey,
thanks for the kind words.
If I'm right your question is the title? If so you can use the `\Spatie\Activitylog\Traits\CausesActivity` trait in your `User` model and after this access all activities by the user via the `actions` relationship.

```php
$user->actions;
```

**Reply by mira-thakkar** [reply]
Yeah thanks a lot! @Gummibeer 
I was finding just by `activities` and finally got `actions` in documentation.
Thanks for help though :)
